doc/recipe/preprocessor.rst

```diff
@@ -325,8 +365,25 @@ experiment is preferred for fx data retrieval:
         weighting_landsea_fraction:
           area_type: land
           exclude: ['CanESM2', 'reference_dataset']
-          fx_variables: [{'short_name': 'sftlf', 'exp': 'piControl'}, {'short_name': 'sftof', 'exp': 'piControl'}]
+          fx_variables: 
+            sftlf:
+              exp: piControl
+            sftof:
+              exp: piControl
 
+or alternatively:
+
+.. code-block:: yaml
+
+    preprocessors:
+      preproc_weighting:
+        weighting_landsea_fraction:
+          area_type: land
+          exclude: ['CanESM2', 'reference_dataset']
+          fx_variables: [
+            {'short_name': 'sftlf', 'exp': 'piControl'}, 
+            {'short_name': 'sftof', 'exp': 'piControl'}
+            ]
```

Comment:
last time I coded up this `fx_variables` dicts (or list of dicts) could accept the same number of data parameters as `dataset` - is this still the case? If so, it's prob good to mention it (unless I missed it)

Reply:
Yes, it's still the case. Otherwise some tests would not be passing, but I can add more examples in the documentation.